stage0/src/main.rs

```diff
@@ -150,9 +150,16 @@ pub extern "C" fn rust64_start(encrypted: u64) -> ! {
     // We assume 0-th bit is never the encrypted bit.
     let encrypted = if encrypted > 0 { 1 << encrypted } else { 0 };
 
+    // Initialize the bump allocator.
+    // Safety: we're the only thread, and we need to explicitly zero out the internal state if we're
+    // running under memory encryption.
+    let alloc = unsafe { BOOT_ALLOC.write(Bump::uninit()) };
+
     // If we're under SEV-ES or SNP, we need a GHCB block for communication.
     let ghcb_protocol = if es {
-        Some(sev::init_ghcb(snp, encrypted))
+        // No point in calling expect() here, the logging isn't set up yet.
```

Comment:
Can you add a comment on what you would write into the `expect` message, though?

Reply:
Done.